scala-package/pom.xml

```diff
@@ -39,6 +39,8 @@
     <scala.version>2.11.8</scala.version>
     <scala.binary.version>2.11</scala.binary.version>
     <build.platform />
+    <cxx>g++</cxx>
+    <dollor>$</dollor>
```

Comment:
Should this be dollar?

Reply:
True, will fix this in next PR with maven build improvement.